src/main/java/io/kestra/plugin/shopify/customers/ListCustomers.java

```diff
@@ -0,0 +1,152 @@
+package io.kestra.plugin.shopify.customers;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.serializers.JacksonMapper;
+import io.kestra.plugin.shopify.AbstractShopifyTask;
+import io.kestra.plugin.shopify.models.Customer;
+import io.kestra.plugin.shopify.models.FetchType;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import jakarta.validation.constraints.NotNull;
+import java.io.ByteArrayInputStream;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "List customers from Shopify store",
+    description = "Retrieve a list of customers from your Shopify store with optional filtering and pagination."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "List all customers",
+            full = true,
+            code = """
+                id: shopify_list_customers
+                namespace: company.team
+                
+                tasks:
+                  - id: list_customers
+                    type: io.kestra.plugin.shopify.customers.ListCustomers
```

Comment:
Please change this to
```suggestion
                    type: io.kestra.plugin.shopify.customers.List
```

Reply:
sure